ansible/roles/cli/tasks/deploy.yml

```diff
@@ -48,10 +50,7 @@
     -C {{ openwhisk_build_dir }}/{{ openwhisk_cli.archive_name }}/
 
 - name: "Copy expanded archive to final configuration directory"
-  copy:
-    #  WARNING:  The trailing slash is significant, signalling to copy contents
-    src: "{{ openwhisk_build_dir }}/{{ openwhisk_cli.archive_name }}/"
-    dest: "{{ openwhisk_cli.nginxdir.name }}"
+  shell: cp -r {{ openwhisk_build_dir }}/{{ openwhisk_cli.archive_name }}/ {{ openwhisk_cli.nginxdir.name }}
```

Comment:
This will not work. If this playbook is run on a remote machine. `copy` does work transparently from local to remote, a shell command won't.